include/deal.II/grid/manifold.h

```diff
@@ -548,7 +548,8 @@ class Manifold : public Subscriptor
 
   /**
    * Return the normal vector to a face embedded in this manifold, at
-   * the point p. If p is not in fact on the surface, but only
+   * the point p. It is not required that the normals actually point
+   * outward. If p is not in fact on the surface, but only
```

Comment:
What does "outward" mean in this context? This function takes a face iterator. Relative to the face, there is should be a criteria where to point to to. However, it the normal does not point outwards of a cell, since there is no notion of a cell here (except at boundary faces).

Reply:
It means the outward of a computational domain,
similar to the description in https://github.com/dealii/dealii/blob/664a66fc6f7a5af7feef246c6c03f86ecb7c3341/include/deal.II/grid/manifold.h#L614

Reply:
@peterrum If you read the rest of the documentation it is pretty clear that a typical usage is a face iterator on the boundary of the domain. I still think this PR is useful information (I was not sure about this fact).